### DISCUSSION TITLE
Allow different files in the same folder to access the same variable?

### DISCUSSION THREAD

**Discussion by DreepyYunky** [odin-lang/Odin#2920] [Ideas/Requests]
This is the idea. If a create a variable/function in `foo.odin`

```odin
x: int = 5
```

Maybe I should be given the ability to access it in `bar.odin`
Like this:

```odin
fmt.println(x)
```

Give me your ideas and opinions.

**Comment by Kelimion** [comment]
That's the default already. Anything within the same package is visible across files and even across packages if imported elsewhere.
